.github/workflows/electron-linux.yml

```diff
@@ -18,9 +18,9 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Setup Node version 🍀
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
```

Comment:
Check if there is a new version for other actions as well:

- actions/checkout@v3 -> v4
- actions/upload-artifact@v3 -> v4

https://github.com/actions/checkout
https://github.com/actions/upload-artifact

Reply:
I add changes in separate commits)